deploy.groovy

```diff
@@ -0,0 +1,74 @@
+//Groovy scripts throw lots of WARNING messages, but it is fine
+//https://issues.apache.org/jira/browse/GROOVY-8339
+
+//Edit this file to define the following Strings
+def SET_DOCKER_REG = "ip:port"
```

Comment:
❓ When would these ever have to be manually edited? Does docker always set these environment properties? I'd assume it does.

Reply:
These are removed.